docs/introduction_tutorial/intro_tutorial_part1.ipynb

```diff
@@ -4,33 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<p>Good catch. I couldn't figure out how to make the sphinx reference to that part work. But the docs are here: https://pangeo-forge.readthedocs.io/en/latest/recipe_user_guide/file_patterns.html#combine-dimensions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I couldn't figure this out right now. I can come back and give it `[testing link]({ref}`recipe_user_guide/file_patterns:combine-dimensions`)` this is the closest I got but I couldn't get it to render. I will try to come back to this tomorrow.

Reply:
As of this morning I added a commit that made the link to the Combine Dimensions route to ConcatDim. It isn't exactly what we want but it it's very close and for the sake of getting this merged ASAP I would vote that it is good enough for now.